config.toml.in

```diff
@@ -7,6 +7,8 @@
 # The storage section controls the configuration of the backend.
 [storage]
 type = "file"
+auto_backup = true
```

Comment:
I suppose a "backup" is a concept of the safe, so it makes sense to set `auto_backup` outside of the `[storage]` table.

I could image setting this at the top level, or in a `safe` or `general` table. I'm leaning toward top level.